browser/ui/tabs/features.cc

```diff
@@ -30,7 +30,7 @@ BASE_FEATURE(kBraveSplitView,
              base::FEATURE_DISABLED_BY_DEFAULT);
 
 bool HorizontalTabsUpdateEnabled() {
-  return base::FeatureList::IsEnabled(kBraveHorizontalTabsUpdate);
+  return true;
```

Comment:
We don't want to check the new feature flag here?

Reply:
No. This is the old feature flag that allowed for the old tabs look, which in this PR is removed entirely. I was thinking that we should address removing this helper in a follow up (in case we want to uplift).